src/lib.rs

```diff
@@ -144,14 +144,20 @@ macro_rules! nonmax {
             /// primitive type.
             #[inline]
             pub const unsafe fn new_unchecked(value: $primitive) -> Self {
-                let inner = core::num::$non_zero::new_unchecked(value ^ $primitive::max_value());
+                let inner = core::num::$non_zero::new_unchecked(value ^ $primitive::MAX);
                 Self(inner)
             }
 
             /// Returns the value as a primitive type.
             #[inline]
             pub const fn get(&self) -> $primitive {
-                self.0.get() ^ $primitive::max_value()
+                self.0.get() ^ $primitive::MAX
+            }
+        }
+
+        impl Default for $nonmax {
+            fn default() -> Self {
+                unsafe { Self(core::num::$non_zero::new_unchecked(0 ^ $primitive::MAX)) }
```

Comment:
```suggestion
                unsafe { Self::new_unchecked(0) }
```

Can we use the existing method so that we can avoid duplicating this code?

Reply:
Sure, I've rebased.